### ISSUE TITLE
Remove OgreXML

### ISSUE THREAD

**Issue by MeFisto94** [jMonkeyEngine/sdk#264]
Maybe as an intermediate solution add a warning dialogue to recommend gltf.

**Comment by tonihele** [comment]
Also does this issue justify removing the million years old bundled Blender from the SDK? I don't see why it would be needed anymore. Was it that the particular version we use just happens to for with the OgleXML stuff? Now that it will be gone. We could remove Blender altogether?

The old version probably can't do GLTF. And I don't see much point in getting a newer version to bloat the SDK side. People would be free to choose their modeling software, and version. Am I right @MeFisto94 ?

**Comment by MeFisto94** [comment]
The reason that we did bundle blender is twofold:
Yes, it is because that used to be some very specific version that was the only compatible with the ogre xml script I think, but we bumped that to 2.79, the last version that the .blend importer supported.

We also use it to invoke scripts on it to be able to import fbx without relying on the engines FBX importer (and maybe other formats such as .dae), which is exactly the same way unity does it, minus the fact that they search Unity on PATH and export to fbx and import fbx manually. We'd need to change that to support more recent blenders and import .dae, .fbx, .stl and what not into blender -> auto convert to gltf and import the gltf into j3o.

The only concern I have that way, is that we loose the ability to support non-standard-pbr-shading materials, because that's the only thing gltf can really express without extensions. But I guess if you're not willing to work in an age old blender version as an artist, that's the only way you got anyway.
